docs/sql-programming-guide.md

```diff
@@ -1843,6 +1843,7 @@ working with timestamps in `pandas_udf`s to get the best performance, see
 
 ## Upgrading From Spark SQL 2.3 to 2.4
 
+  - Since Spark 2.4, Spark will display hive table description column  `Last Access` value  as `UNKNOWN` following the Hive system.
```

Comment:
This is applicable to both native and hive tables. How about changing it to 

> Spark will display table description column  `Last Access` value  as `UNKNOWN` when the value was `Jan 01 1970`.

Reply:
Right, its applicable for both type, i will update the message as per your comment. Thanks

Reply:
done.